src/CMakeLists.txt

```diff
@@ -380,6 +380,7 @@ if(WITH_SEASTAR)
   endmacro ()
   set(Seastar_API_LEVEL "6" CACHE STRING "" FORCE)
   set(Seastar_HWLOC OFF CACHE BOOL "" FORCE)
+  set(Seastar_IO_URING OFF CACHE BOOL "" FORCE)
```

Comment:
an alternative is to update `cmake/modules/Finduring.cmake`, so it is able to expose `VERSION_VAR`, so that `seastar/cmake/SeastarDependencies.cmake` is able to check for the required version properly.

Reply:
i see. `seastar/cmake/FindLibUring.cmake` did its job and found the liburing 2.0 and up installed in system. but somehow, the liburing built from source won the game, and gets included, and then failed the build.

Reply:
@rosinL Rixin, do you know why `uring::uring` is linked by Seastar?